### DISCUSSION TITLE
Does it compatible with .net 8 version?

### DISCUSSION THREAD

**Discussion by srini146** [reqnroll/Reqnroll#163] [Q&A] [answered]
Please confirm if ReqNroll is compatible with .NET 8, as we're considering migrating from SpecFlow. Additionally, I'm interested in understanding how ReqNroll supports SpecFlow's living documentation.

**Comment by ajeckmans** [comment, marked as answer]
It is compatible. I use it in several .net 8 projects.

The livingdoc is currently not supported, see #15.

**Comment by Napoleon10** [comment]
I will second @ajeckmans comments, I am using .NET 8 with ReqNroll. I had 15 automation projects using SpecFlow, Visual Studio and DevOps. There is information in ReqNrolls documentation of the migration process. What is this item NOT in 'Answered'?
